### ISSUE TITLE
Add Projects in ML section

### ISSUE THREAD

**Issue by IshikaBanga26** [pavitraag/Project-Vault#257]
I would like to contribute two of my Machine Learning projects to the **ML section** of the ProjectVault website:

 **1. Object Classification**

* Overview: A CNN-based project that classifies objects like numbers in given image.
* Tech Stack: Python, TensorFlow/Keras, OpenCV
* Features:

  * Image preprocessing
  * Model training & validation
  * High accuracy on test set

 **2. Landmark Detection**

* Overview: A classification model that identifies famous geographical landmarks from images (e.g., Eiffel Tower, Taj Mahal, Statue of Liberty).
* Tech Stack: Python,  / TensorFlow/Keras,  /OpenCV
* Features:

  * Real-time detection
  * Image-Based Landmark Recognition
  * High Accuracy

I would love to integrate both projects into the ML section of Project Vault. I can create detailed project cards and contribute the code, README, and images as needed.

Please let me know if I can proceed with a PR!

**Comment by pavitraag** [comment]
I gave level 2 as u r creating two projects in one issue @IshikaBanga26
